user_management/doc/User_management.md

```diff
@@ -652,3 +652,79 @@ all fields are mandatory
 > | `500`     | `application/json` | `{"errors": ['An unexpected error occurred : ...']}` |
 
 </details>
+
+## `/user/avatar/`
+
+This endpoint allows the user to get and update his avatar.
+
+### Get user's avatar
+
+<details>
+ <summary><code>GET</code><code><b>/user/avatar/</b></code></summary>
+
+### Parameters
+
+Authorization: {access_token}
+
+NB : any access_token can be used to get the avatar of any user
```

Comment:
The documentation should be updated on this point, as it is no longer the case.

Reply:
[400a648](https://github.com/tdameros/42-transcendence/pull/210/commits/400a648f8a0c55b937561a18e61d914d62266666)